Library/AppEnvironment.swift

```diff
@@ -71,12 +72,13 @@ public struct AppEnvironment {
   }
 
   // Push a new environment onto the stack.
-  public static func pushEnvironment(env: Environment) {
+  public static func pushEnvironment(_ env: Environment) {
     saveEnvironment(environment: env, ubiquitousStore: env.ubiquitousStore, userDefaults: env.userDefaults)
     stack.append(env)
   }
 
   // Pop an environment off the stack.
+  @discardableResult
   public static func popEnvironment() -> Environment? {
     let last = stack.popLast()
```

Comment:
we've actually never made use of the return value of `popEnvironment` so we might want to just change this to a void function...